common/predictive-text/unit_tests/headless/worker-custom-punctuation.js

```diff
@@ -8,16 +8,7 @@ var ModelCompositor = require('../../build/intermediate').ModelCompositor;
 
 describe('Custom Punctuation', function () {
   it('appears in the keep suggestion', function () {
-    let dummySuggestions = [
-      {
-        transform: {
-          insert: 'Hrllo',
-          deleteLeft: 0,
-        },
-        tag: 'keep',
-        displayAs: 'Hrllo',
-      },
-      {
```

Comment:
The 'keep' suggestion is automatically generated by the compositor.  If the model returns it instead, the compositor assumes it to be an actual prediction.

This only resulted in a breakage now due to the new changes, though the test was ineffective for a while before this.